lib/core/certificates/step.sh

```diff
@@ -53,9 +53,10 @@ function ih::setup::core.certificates::install() {
 
   ih::log::info "Acquiring cert bundle from Mozilla"
   curl https://curl.se/ca/cacert.pem >"$MOZILLA_PATH"
-
-  curl https://curl.se/ca/cacert.pem >"$HOME/.ih/certs/mozilla.pem"
+  # Append our DLP cerTs to the mozilla bundle.
   cat "$HOME/.ih/certs/grand_rounds_chained_ca.pem" >>"$HOME/.ih/certs/mozilla.pem"
```

Comment:
I'm a little confused by the different variables in this file. Is the file being modified here the same as `$MOZILLA_PATH`? Is this line the same as L62?

Reply:
Yeah, looks like this line should be replaced by L62, and L62 can be deleted. Also looks like L60 can be changed to reference `$MOZILLA_PATH`:
```
cat "$CA_PATH" >>"$MOZILLA_PATH"
npm config set cafile "$MOZILLA_PATH"
```

Reply:
Tidied this up.